### ISSUE TITLE
'Invalid range' error when applying filter commands

### ISSUE THREAD

**Issue by Jorenar** [rickhowe/wrapwidth#12]
If we filter buffer with `:%!cat`, the following error message will be shown:
```
Error detected while processing TextChanged Autocommands for "<buffer=1>"..function <SNR>88_CheckEvent[61]..<SNR>88_Proplist:
line    1:
E16: Invalid range
```

**Comment by rickhowe** [comment]
Hi, I also fixed this issue on 3.3.

**Comment by Jorenar** [comment]
Thanks! :)
